File: uri/salario/salario.py
def calculasalario( a, b ):
    return (a * b)

num = int(input())
tempo = int(input())
pagportempo = float(input())

foo = calculasalario(tempo, pagportempo)
print "NUMBER =",num
print "SALARY = U$ {0:.2f}".format(foo)

"""
def calculasalario( a, b ):
    return (a * b)

num = int(input())
tempo = int(input())
pagportempo = float(input())

foo = calculasalario(tempo, pagportempo)
print ("NUMBER =",num)
print ("SALARY = U$ {0:.2f}".format(foo))

"""

File: uri/salario/salario.rb
def calculasalario( a, b )
    return (a * b)
end

num = gets.to_i
tempo = gets.to_i
pagportempo = gets.to_f

foo = calculasalario(tempo, pagportempo)
puts "NUMBER = %d" % [num]
puts "SALARY = U$ %.2f" % [foo]